packages/amplify-graphql-api-construct-tests/src/__tests__/relationships/composite-pk/relationships-cpk.test.ts

```diff
@@ -80,6 +81,10 @@ describe('Relationships defined using composite primary are supported', () => {
       test('Associated models included in query and mutation response with two sort keys', async () => {
         await testPrimaryCpkSkTwoContainAssociated(currentId, apiEndpoint, apiKey);
       });
+
+      test('Multiple RelatedOne associated records returns one RelatedOne record', async () => {
+        await testPrimaryMultipleHasOneContainsOneHasOne(currentId, apiEndpoint, apiKey);
```

Comment:
We should add this test for SQL sources as well, to ensure no regressions in future

Reply:
Good call. Addressed in [0e53404](https://github.com/aws-amplify/amplify-category-api/pull/2497/commits/0e53404940397c520d44ddea6495d0d6fb738232)